configure.ac

```diff
@@ -32,12 +32,12 @@ AC_CACHE_SAVE
 LT_INIT([shared disable-static dlopen])
 
 dnl Work out the C++ standard which we are going to use
-AX_CXX_COMPILE_STDCXX_1Y
-AX_CXX_COMPILE_STDCXX_0X
+AX_CXX_COMPILE_STDCXX_17(noext)
 
-AS_IF([test "x$found_cxx1y" = "xyes"],
-        [CXXFLAGS="$CXXFLAGS $SST_CXX1Y_FLAGS"],
-        [CXXFLAGS="$CXXFLAGS $SST_CXX0X_FLAGS"])
+CXXFLAGS="$CXXFLAGS -std=c++17"
```

Comment:
I think you should just leave this line out. I believe the macro will just redefine CXX (CXX="$CXX $switch") and will detect different switches for different compilers. Probably not a problem for the compilers we support, but it's the way it's intended to work. (Do your compile commands have duplicate -std=c++17?)

Reply:
That flag is required there but I am removing -std=c++17 from CXX so it's cleaner.